### PR TITLE
Ports tg embed and pellet clouds

### PR DESCRIPTION
**Pull request by jupyterkat**
## About The Pull Request

Adds sticky tapes and pellet cloud grenades
A sticky apocalypse adminbuse event
Ports tgstation/tgstation#49634 and supporting prs

## Why It's Good For The Game

yes

## Changelog
:cl: Ryll-Ryll
add: Added stickytapes
add: Added pellet cloud grenades
refactor: Refactored embeds, there can now be harmless embeds
add: Added back the pellet cloud option to admin podlaunchers
/:cl:

### CODE REVIEW

**Comment by jupyterkat** [comment]
image

**Comment by utilbird** [comment]
TAPE

File: code/game/objects/items/grenades/syndieminibomb.dm [hunk 1]
```diff
@@ -4,27 +4,45 @@
...
+	desc = "An anti-personnel fragmentation grenade, this weapon excels at killing soft targets by shredding them with metal shrapnel."
 	icon_state = "frag"
+	shrapnel_type = /obj/item/projectile/bullet/shrapnel
+	shrapnel_radius = 4
+	ex_heavy = 1
+	ex_light = 3
+	ex_flame = 4
+
+/obj/item/grenade/frag/mega
+	name = "FRAG grenade"
```
Comment (utilbird) [inline code/game/objects/items/grenades/syndieminibomb.dm]:
```suggestion
	name = "\improper FRAG grenade"
```

File: code/game/objects/items/stacks/tape.dm [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	max_amount = 5
+
+	var/list/conferred_embed = EMBED_HARMLESS
+	var/overwrite_existing = FALSE
+
+/obj/item/stack/sticky_tape/afterattack(obj/item/I, mob/living/user)
+	if(!istype(I))
+		return
+
+	if(I.embedding && I.embedding == conferred_embed)
```
Comment (utilbird) [inline code/game/objects/items/stacks/tape.dm]:
```suggestion
	if(I.embedding == conferred_embed)
```

**Review by utilbird — APPROVED** [review]

**Pull request merged**